contrib/pipelines/secadvisor/README.md

```diff
@@ -2,33 +2,223 @@
 
 Implements the IBM Security Advisor pipeline
 
+The security advisor filters the flow data obtained from skydive, performs a data transformation, and saves the information to an object store in JSON format.
+This data may then be used to perform various kinds of analyses for security, accounting, or other purposes.
+We use the term "pipeline" to describe the sequence of operations performed on the data: classify, filter, encode, store.
```

Comment:
I suggest you remove this line - an explanation about pipelines will come (later) at the top-level or core